salt/metalk8s/volumes/init.sls

```diff
@@ -0,0 +1,2 @@
+include:
+  - .prepare
```

Comment:
Let's try to fix this early: formulas are usually named as describing a **state**, not an **action**. So that would be `prepared.sls`, and could be accompanied by an `unprepared.sls` counterpart.

Reply:
Ok, will rename it to `prepared`.

As for `unprepared` it's is planned (#1475)